chapters/object_types/frame.txt

```diff
@@ -15,7 +15,10 @@ ANARIFrame anariNewFrame(ANARIDevice);
 ....
 
 The frame uses parameters to encode size, color format, and which
-channels to use.
+channels to use. Channels are identified by string names with a `channel.` prefix. The frame
+object has an identically named parameter for each reported channel. Setting one of the allowed
+data types enables the channel and determines the data type it can be mapped as. The same
+prefixed channel names are used to select the channel in `anariMapFrame`.
```

Comment:
In the discussion, summarized in https://github.com/KhronosGroup/ANARI-Docs/issues/37#issuecomment-1211076350, we said `mapFrame` gets the channel name (returned by the `channel` info query), without prefix.

I still a bit wary about those inconsistencies: users will often need to go the the spec to check whether _here_ a prefix is needed or not. Can we treat the prefix as a naming convention and just have it everywhere (accepting the redundancy where it is "clear" that a channel is meant, like for `mapFrame` or the `channel` info query). Which also means the `channel` info query is "consistent" to the `parameter` info query. Then also apps can use the name strings without any string processing.

Reply:
I thought we went with using the same names everywhere (with the prefix) and with that "bias" I read the summary as also saying that. I guess a pedantic reading of the summary says channel names for `mapFrame` are the same as returned by the queries and then only the part about parameters talks about the prefix. But it's unclear if that is meant to imply the others don't have the prefix?

Reply:
I think the latest changes satisfy the consistency + clarity we discussed at the WG meeting. I think this is ready to merge.